### PR TITLE
Append new authorized user to blossom-ci safelist

### PR DESCRIPTION
**Pull request by amahussein**
adding authorized github user to the safelist.

### CODE REVIEW

File: .github/workflows/blossom-ci.yml [hunk 1]
```diff
@@ -64,6 +64,7 @@ jobs:
       res-life,\
```
Comment (jlowe) [inline .github/workflows/blossom-ci.yml:64]: 2022 copyrights

**Review by jlowe — COMMENTED** [review]
Commit needs a signoff per the [contributor guidelines](https://github.com/NVIDIA/spark-rapids/blob/branch-22.04/CONTRIBUTING.md#sign-your-work).  You can do this on any commit in the PR, so that can be fixed in the copyright update commit.

**Comment by jlowe** [comment]
build

**Comment by amahussein** [comment]
build

**Comment by andygrove** [comment]
build failed with:

```
Error: 1-28T17:38:28.398Z] [ERROR] Failed to execute goal on project rapids-4-spark-sql_2.12: Could not resolve dependencies for project com.nvidia:rapids-4-spark-sql_2.12:jar:22.04.0-SNAPSHOT: Could not find artifact log4j:log4j:jar:1.2.17-cloudera1 in apache-snapshots-repo (https://****/artifactory/sw-spark-maven) -> [Help 1]
```

**Comment by mythrocks** [comment]
> `log4j:log4j:jar:1.2.17-cloudera1`

Cloudera's [response to Log4Shell](https://blog.cloudera.com/cloudera-response-to-cve-2021-44228/), most likely.

**Comment by tgravescs** [comment]
build

**Comment by mythrocks** [comment]
@amahussein Congratulation! 🎉

**Pull request merged**